runtime/src/asset.rs

```diff
@@ -1035,6 +1055,26 @@ decl_module! {
             Self::deposit_event(RawEvent::CustodyTransfer(ticker.clone(), custodian_did, holder_did, receiver_did, value));
             Ok(())
         }
+
+        /// Sets the name of the current funding round.
+        ///
+        /// # Arguments
+        /// * `origin` - the signing key of the token owner DID.
+        /// * `did` - the token owner DID.
+        /// * `ticker` - the ticker of the token.
+        /// * `name` - the desired name of the current funding round.
+        pub fn set_funding_round(origin, did: IdentityId, ticker: Vec<u8>, name: Vec<u8>) -> Result {
+            let ticker = utils::bytes_to_upper(ticker.as_slice());
+            let sender = ensure_signed(origin)?;
```

Comment:
very minor, but usually the ensure_signed would be the first thing checked in a function I think.